hibernate-core/src/test/java/org/hibernate/test/schemaupdate/PostgreSQLMultipleSchemaSequenceTest.java

```diff
@@ -48,6 +50,7 @@
  * @author Vlad Mihalcea
  */
 @RequiresDialect(PostgreSQL81Dialect.class)
+@RequiresDialectFeature(value = DialectChecks.SupportSchemaCreation.class)
```

Comment:
Not a big deal, but sometimes you omit the `value` spec elsewhere. A little bit inconsistent, but I think we don't have coding style guide on this.

Reply:
I removed `value` when there is only one argument.